### PR TITLE
Profiler update for jwtAccess Tokens

### PR DESCRIPTION
**Pull request by XerTheSquirrel**
This is an update to the profiler so that it can report jwtAccess tokens.

This is currently in progress.

There is going to be an incompatible change which will be performed in the `IOpipeMeasurement` class and that will be the removal of `buildRequest()`. That method should not have been in that class and it should not have been public anyway, the class should just store some measurement information and such accordingly, not perform much more work on building requests. This should have no impact at all because there is really no need for the user to build requests when that is automatically done by the service.

### CODE REVIEW

**Comment by XerTheSquirrel** [comment]
Okay so currently I believe these are all the needed changes to use the new key values for uploaded profiler data.

**Comment by XerTheSquirrel** [comment]
Okay so this works in the staging dashboard now and I believe that it is ready to be reviewed for merging.

File: src/main/java/com/iopipe/IOpipeExecution.java [hunk 1]
```diff
@@ -351,5 +370,333 @@ public final ThreadGroup threadGroup()
 	{
 		return this.threadgroup;
 	}
+
+	/**
+	 * Builds the request which is sent to the remote service.
```
Comment (pselle) [inline src/main/java/com/iopipe/IOpipeExecution.java]: I have to look at this in full context but ... this looks like a duplicate add? I see the PR notes about removing buildRequest() so shouldn't there be a deletion here?

Comment (XerTheSquirrel) [reply src/main/java/com/iopipe/IOpipeExecution.java]: Oh, I meant from a user perspective the method is now gone. The code there is still useful and I just move it to where it is better located and make it hidden so the user cannot access it (they have no need to anyway).

Comment (pselle) [reply src/main/java/com/iopipe/IOpipeExecution.java]: Ah got it, I was looking for a deletion, and the whole file was removed 👍 Got it

**Comment by XerTheSquirrel** [comment]
Also, I did not re-generate the documentation for this at all because of the inconsistency with it. It would have complicated the diff (by having a bunch of HTML pages around), but the documentation can be updated on the next release however (it will have to be anyway due to the constant changes).

**Pull request merged**